Cooper.DAO/ChatDAO/ChatDAO.cs

```diff
@@ -0,0 +1,220 @@
+﻿using System;
+using Cooper.DAO.Mapping;
+using Cooper.DAO.Models;
+using Cooper.ORM;
+using Cooper.Services.Interfaces;
+using NLog;
+using System.Collections.Generic;
+using System.Linq;
+using Oracle.ManagedDataAccess.Client;
+using System.Data.Common;
+
+namespace Cooper.DAO
+{
+    public class ChatDAO : IChatDAO
+    {
+        private readonly CRUD crud;
+        private readonly Logger logger;
+        private readonly DbConnect dbConnect;
+
+        private string table;
+        private string idColumn;
+        private HashSet<string> attributes;
+
+        public ChatDAO(IConfigProvider configProvider)
+        {
+            crud = new CRUD(configProvider);
+            logger = LogManager.GetCurrentClassLogger();
+            dbConnect =  new DbConnect(configProvider);
+
+            table = "CHATS";
+            idColumn = "ID";
+            attributes = new HashSet<string>()
+            {
+                "ID", "CHATNAME", "PHOTOURL"
+            };
+        }
+        
+        public IList<ChatDb> GetPersonalChats(long userId)
+        {
+            IList<ChatDb> personalChats = null;
+
+            var chatAttributes = new HashSet<string>()
+            {
+                "ID"
+            };
+
+            string sqlExpression = String.Format("SELECT IDCHAT as ID FROM {0} INNER JOIN USERSCHATS ON {0}.ID = USERSCHATS.IDCHAT WHERE IDUSER = {1}", table, userId);
```

Comment:
Please, check performance of this query and something like this:
select IDCHAT as ID from {0} where exists (select 1 from USERSCHATS where {0}.ID = USERSCHATS.IDCHAT AND IDUSER = {1}

You can use "Explain plan" in sql developerr